docs/modules/ROOT/pages/how-tos/testing/integration/choosing-contexts.adoc

```diff
@@ -0,0 +1,66 @@
+= Choosing when to run certain Integration Tests
+
+Integration test scenarios can be configured to run in only certain cases (referred to as contexts in further text).
+Examples include running the integration test pipelines only in case of Pull Requests, or running them only for builds of specific components etc.
```

Comment:
Nit: remove the `etc` as we are not trying to say that this is an exhaustive list of examples.

Reply:
Ack, removed it.